src/rgw/rgw_file.h

```diff
@@ -456,11 +456,6 @@ namespace rgw {
 	st->st_nlink = state.nlink;
 	break;
       case RGW_FS_TYPE_FILE:
-	st->st_nlink = 1;
```

Comment:
I don't see how we want to do this

Reply:
right. will revert

Reply:
in the original version `st->st_link = 1;` was under both `RGW_FS_TYPE_FILE` and `RGW_FS_TYPE_SYMBOLIC_LINK`
so, either we can collapse them to one case, or we need to fix this line in one of the cases

Reply:
In this case, we're kind of inventing a value for reporting purposes, but I'd like them to be distinct for the different object types for clarity.

Reply:
originally, there were 2 different cases with exactly the same code.
so, either, we can collapse them to one case, or there was a copy&paste bug in the original code.
but if you think it is ok to duplicate the code in the 2 cases, i can revert the change

Reply:
yes, there is no problem having 2 cases that take the same action, in this case

Reply:
ok. will revert